templates/zabbix_server.conf.erb

```diff
@@ -77,6 +77,17 @@ DebugLevel=<%= @debuglevel %>
 #
 PidFile=<%= @pidfile %>
 
+<% if @zabbix_version.to_f >= 3.4 %>
+### Option: SocketDir
+#       IPC socket directory.
+#               Directory to store IPC sockets used by internal Zabbix services.
+#
+# Mandatory: no
+# Default:
+
+SocketDir=<%= @socket_dir %>
```

Comment:
```suggestion
SocketDir=<%= @socketdir %>
```

Reply:
I evidently missed this in my internal refactoring, because I originally had the parameter as socket_dir rather than socketdir. Would also explain the failing tests!

Reply:
Fixed in latest commit.